deploy/examples/create-external-cluster-resources.py

```diff
@@ -399,6 +399,12 @@ def gen_arg_parser(cls, args_to_parse=None):
             required=False,
             help="Ceph Manager prometheus exporter port",
         )
```

Comment:
Add a specific CI test for the same, and also the unit test
https://github.com/rook/rook/blob/master/.github/workflows/canary-integration-test.yml#L74,
https://github.com/rook/rook/blob/master/deploy/examples/create-external-cluster-resources-tests.py

And also update the doc.
https://github.com/rook/rook/blob/master/Documentation/CRDs/Cluster/external-cluster.md

Reply:
Hi @parth-gr, thanks for the review! I have updated the PR with unit tests, a simple dry run to test that the flag is accepted, as well as a line in the documentation.

I've tried to follow the manners of the surrounding code wherever possible, please let me know if more changes are required, thanks!